src/test/java/com/spotify/futures/CompletableFuturesTest.java

```diff
@@ -429,4 +431,23 @@ public void combine5_exceptional() throws Exception {
     exception.expectCause(isA(IllegalStateException.class));
     getCompleted(future);
   }
+
+  private static <T> Matcher<CompletionStage<T>> completesTo(final T expected) {
+    return completesTo(is(expected));
+  }
+
+  private static <T> Matcher<CompletionStage<T>> completesTo(final Matcher<T> expected) {
+    return new CustomTypeSafeMatcher<CompletionStage<T>>("complete to" + String.valueOf(expected)) {
+      @Override
+      protected boolean matchesSafely(CompletionStage<T> item) {
+        try {
+          final T value = item.toCompletableFuture().get(1, TimeUnit.SECONDS);
+          return expected.matches(value);
+        } catch (Exception ex) {
```

Comment:
The current failure in case of a timeout reads like this:
```
java.lang.AssertionError: 
Expected: completes to is "hello"
     but: was <java.util.concurrent.CompletableFuture@4361bd48[Not completed]>
```
I considered that good enough for now, but I'm open for ideas (separate PR?)